# 3D bounding box array message
# The idea is from
# https://github.com/ros-perception/vision_msgs/blob/noetic-devel/msg/BoundingBox3DArray.msg

# The actual bounding boxes
BoundingBox3D[] boxes
